### PR TITLE
PMCP-1746: Add Deadline tokens

### PR DESCRIPTION
**Pull request by yahilmadakiya**

### CODE REVIEW

File: packages/larva-tokens/src/base/aliases.json [hunk 1]
```diff
@@ -45,6 +46,8 @@
 		"IBM_PLEX_SERIF": "'IBM Plex Serif', serif",
 		"IBM_PLEX_MONO": "'IBM Plex Mono', monospace",
 		"PARA_SUPREME": "'Para Supreme Regular', serif",
-		"GRAPHIK_COND": "'Graphik XX Cond', sans-serif"
+		"GRAPHIK_COND": "'Graphik XX Cond', sans-serif",
+		"BALTO": "'Balto, Helvetica, sans-serif",
```
Comment (laras126) [inline packages/larva-tokens/src/base/aliases.json]: Need to close `'`

File: packages/larva-tokens/src/base/aliases.json [hunk 2]
```diff
@@ -45,6 +46,8 @@
...
+		"ELSELL": "'Eksell, Georgia, serif"
```
Comment (laras126) [inline packages/larva-tokens/src/base/aliases.json]: Typo, should be `EKSELL`

File: packages/larva-tokens/src/base/aliases.json (hunk 1 above)
Comment (laras126) [reply packages/larva-tokens/src/base/aliases.json]: (on Balto and Eksell)

File: packages/larva-tokens/src/brands/deadline.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+			"comment": "Available via Typekit."
+		},
+		"FONT_FAMILY_PRIMARY": {
+			"value": "{!HELVETICA}",
+			"type": "font-family",
+			"category": "font-family",
+			"comment": "Fallback for primary-fancy."
+		},
+		"FONT_FAMILY_SECONDARY_FANCY": {
+			"value": "{!ELSELL}",
```
Comment (laras126) [inline packages/larva-tokens/src/brands/deadline.json]: Update to EKSELL when renamed

File: packages/larva-tokens/src/brands/deadline.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+			"comment": "Available via @font-face."
+		},
+		"FONT_FAMILY_SECONDARY": {
+			"value": "{!GEORGIA}",
+			"type": "font-family",
+			"category": "font-family",
+			"comment": "Fallback for secondary-fancy"
+		},
+		"FONT_FAMILY_BODY": {
+			"value": "{!HELVETICA}",
```
Comment (laras126) [inline packages/larva-tokens/src/brands/deadline.json]: FONT_FAMILY_BODY should be Georgia - refer to JIRA ticket

File: packages/larva-tokens/style-guides/deadline.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (laras126) [inline packages/larva-tokens/style-guides/deadline.json]: This looks like an extra file generated - can you delete it? Sometimes the script does that, I'm not sure why.

File: packages/larva-tokens/build/deadline.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (laras126) [inline packages/larva-tokens/build/deadline.json]: This is also an extra file, please delete – the only deadline.json should be in src/brands/

**Comment by yahilmadakiya** [comment]
@laras126 Feedbacks are resolved, Please have a look.

**Pull request merged**